pybamm/batch_study.py

```diff
@@ -86,6 +86,7 @@ def __init__(
         self.repeats = repeats
         self.permutations = permutations
         self.quick_plot = None
+        self.sims = []
```

Comment:
Initializing this in `__init__` would be redundant. We can keep this in its original place.

Reply:
and using `hasattr` will still work